internal/store/model/resource.go

```diff
@@ -56,3 +57,8 @@ func (r *Resource) BeforeCreate(tx *gorm.DB) error {
 	}
 	return nil
 }
+
+func GetResourceVersion(updatedAt time.Time) *string {
+	resourceVersion := base64.StdEncoding.EncodeToString([]byte(updatedAt.String()))
```

Comment:
In terms of performance since you already have time.Time object, another option would be using unix time.
```go
resourceVersion := strconv.FormatInt(updatedAt.UnixNano(), 10)
```

Reply:
I like.  Done.